src/main/java/com/lotto/domain/LottoReward.java

```diff
@@ -0,0 +1,36 @@
+package com.lotto.domain;
+
+public enum LottoReward {
+    THREE(5000) {
+        @Override
+        int totalReward(int count) { return count * THREE.reward(); }
+    },
```

Comment:
```suggestion
    THREE(5_000, count -> count * THREE.reward),
```
이렇게 람다 표현식을 사용해서 표현해보면 어떨까요?